apps/nextjs/src/app/api/chat/errorHandling.test.ts

```diff
@@ -88,14 +96,19 @@ describe("handleChatException", () => {
 
       expect(response.status).toBe(200);
 
-      const consumed = await consumeStream(response.body as ReadableStream);
+      invariant(
+        response.body instanceof ReadableStream,
+        "Expected response.body to be a ReadableStream",
+      );
+
+      const consumed = await consumeStream(response.body);
       const message = extractStreamMessage(consumed);
 
       expect(message).toEqual({
         type: "error",
         value: "Rate limit exceeded",
         message:
-          "**Unfortunately you’ve exceeded your fair usage limit for today.** Please come back in 1 hour. If you require a higher limit, please [make a request](https://forms.gle/tHsYMZJR367zydsG8).",
+          "**Unfortunately you've exceeded your fair usage limit for today.** Please come back in 1 hour. If you require a higher limit, please [make a request](https://forms.gle/tHsYMZJR367zydsG8).",
```

Comment:
I put in this typographic apostrophe. What's the story around making it a "dumb" quote?